File: dongyimai_sellergoods_interface/src/main/java/com/offcn/sellergoods/service/SellerService.java
package com.offcn.sellergoods.service;

import com.offcn.entity.PageResult;
import com.offcn.pojo.TbSeller;

import java.util.List;

public interface SellerService {

	//findAll
	public List<TbSeller> findAll();
	
	//findPage
	public PageResult findPage(int pageNum, int pageSize);
	
	//add
	public void add(TbSeller seller);
	
	//update
	public void update(TbSeller seller);
	
	//findOne
	public TbSeller findOne(String sellerId);

	//delete
	public void delete(String[] sellerIds);

	//条件、模糊查询+分页
	public PageResult findPage(TbSeller seller, int pageNum, int pageSize);

	/**
	 * 商家审核
	 * @param sellerId  商家ID
	 * @param status	审核状态
	 */
	public void updateStatus(String sellerId, String status);
	
}

File: dongyimai_manager_web/src/main/webapp/js/service/sellerService.js
//服务层
app.service('sellerService',function($http){
	    	
	//findAll
	this.findAll=function(){
		return $http.get('../seller/findAll.do');		
	}

	//findPage
	this.findPage=function(page,rows){
		return $http.get('../seller/findPage.do?page='+page+'&rows='+rows);
	}

	//findOne
	this.findOne=function(sellerId){
		return $http.get('../seller/findOne.do?sellerId='+sellerId);
	}

	//add
	this.add=function(entity){
		return  $http.post('../seller/add.do',entity );
	}

	//update
	this.update=function(entity){
		return  $http.post('../seller/update.do',entity );
	}

	//dele
	this.dele=function(ids){
		return $http.get('../seller/delete.do?ids='+ids);
	}

	//条件、模糊查询+分页
	this.search=function(page,rows,searchEntity){
		return $http.post('../seller/search.do?page='+page+"&rows="+rows, searchEntity);
	}

	//商家审核
	this.updateStatus = function(sellerId,status){
		return $http.get('../seller/updateStatus.do?sellerId='+sellerId+"&status="+status);
	}

});